shotover-proxy/src/server.rs

```diff
@@ -99,6 +99,9 @@ pub struct TcpCodecListener<C: Codec> {
 
     /// Keep track of how many messages we have received so we can use it as a request id.
     message_count: u64,
+
+    /// Metric recording shotover available connections
+    gauge: Gauge,
```

Comment:
Can we name the metrics properly rather than putting the name in a comment?
Something like: `available_connections_gauge: Gauge`